Sources/SourceControl/GitRepository.swift

```diff
@@ -741,4 +741,12 @@ private class GitFileSystemView: FileSystem {
     func chmod(_ mode: FileMode, path: AbsolutePath, options: Set<FileMode.Option>) throws {
         throw FileSystemError.unsupported
     }
+
+    func copy(from sourcePath: AbsolutePath, to destinationPath: AbsolutePath) throws {
+        throw FileSystemError.unsupported
```

Comment:
I think we should also add an assertion here since this is never expected to be supported